create-k8s-chained-sessions.sh

```diff
@@ -8,7 +8,6 @@
 
 # global variables
 declare PROFILE_ID
-declare CHAINED_SESSION_IDS="name,id\n"
```

Comment:
I don't think we ever used this output so I removed it.

Reply:
I'm fine with removing it if you want, but I will say I put this in here as a convenience output to transpose specific session ids to my `.zshenv` so I can more easily start specific sessions. That might be references in panopedia as something you can do as well

Reply:
I double checked panopedia and couldn't find any references to this in the active pages we have, so we should be fine there